dnddiceroller/DiceFileReader.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dnddiceroller;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.ArrayList;
+import java.util.List;
+
+
+
+/**
+ *
+ * @author Paul Scarrone
+ */
+public class DiceFileReader {
+  private String filePath;
+  private Scanner input = null;
+  private List<String> lines;
+
+  /*
+    Use this with
+	DiceFileReader reader = new DiceFileReader("dice.txt")
+  */
+  
+  public DiceFileReader(String filePath) {
+	this.filePath = filePath;
+	
+	File inputFile = new File(filePath);
+	
+	try {
+	  this.input = new Scanner(inputFile);
+	} catch (FileNotFoundException ex) {
+	  Logger.getLogger(DiceFileReader.class.getName()).log(Level.SEVERE, null, ex);
+	}
+  }
+  
+  public List<String> getLines(){
+	return this.lines;
+  }
+  
+  public void read(){
```

Comment:
That sounds like a good plan

On Mon, Nov 9, 2015, 13:19 Curtkunklegithubber <notifications@github.com>
wrote:

> In dnddiceroller/DiceFileReader.java
> <https://github.com/WCCCEDU/CPT-163-27-F2015-Assignment-9-Files-Strings/pull/2#discussion_r44309337>
> :
>
> > +	this.filePath = filePath;
> > +	
> > +	File inputFile = new File(filePath);
> > +	
> > +	try {
> > +	  this.input = new Scanner(inputFile);
> > +	} catch (FileNotFoundException ex) {
> > +	  Logger.getLogger(DiceFileReader.class.getName()).log(Level.SEVERE, null, ex);
> > +	}
> > +  }
> > +
> > +  public List<String> getLines(){
> > +	return this.lines;
> > +  }
> > +
> > +  public void read(){
>
> I see what you mean.... What if the read method returned an array list of
> dice?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WCCCEDU/CPT-163-27-F2015-Assignment-9-Files-Strings/pull/2/files#r44309337>
> .
>